gym/spaces/__init__.py

```diff
@@ -1,3 +1,14 @@
+"""This module implements various spaces.
+
+Spaces describe mathematical sets and are used in Gym to specify valid actions and observations.
+Every Gym environment must have the attributes ``action_space`` and ``observation_space``.
+If, for instance, three possible actions (0,1,2) can be performed in your environment and observations
+are vectors in the two-dimensional unit cube, the environment code may contain the following two lines:
+
+::
```

Comment:
I think this should be `Example::`

Reply:
I actually wanted to do `... the following two lines::`. I think that should work as well and I fixed it.